### DISCUSSION TITLE
New YouTube

### DISCUSSION THREAD

**Discussion by Charlieis1nsan3** [3kh0/ext-remover#1398] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
I made a YouTube I will be posting Exploits and stuff I find. As well as tutorials. https://www.youtube.com/@CharlieIsInsaneYT

**Comment by OPcoolman** [comment]
K

**Reply by Luigi-Master** [reply]
I subbed, :)

**Reply by Charlieis1nsan3** [reply]
thanks!

**Comment by Sam-hacks1** [comment]
can you make a vid showing how to do snap and run

**Reply by Charlieis1nsan3** [reply]
sure.

**Reply by Sam-hacks1** [reply]
actually do you know how to do an exploit from read & write because @schoolexploitkid said there is one?

**Comment by OPcoolman** [comment]


**Reply by Charlieis1nsan3** [reply]
yeah I noticed that, I fixed it, you should be able to see the full link now.
